helpers/env.go

```diff
@@ -50,3 +50,7 @@ func GetEnvBool(key string, fallback bool) bool {
 		return parsedValue, err
 	}).(bool)
 }
+
+func IsDebugEnv() bool {
```

Comment:
Other options include adding an env var for [prod, sandbox, testing] but I don't think we're there yet as a company

Reply:
good call out, and something to remember in the near future.